scripts/lib/setup-apt-repo

```diff
@@ -37,14 +37,24 @@ SCRIPTS_PATH="$(dirname "$(dirname "$0")")"
 
 release=$(lsb_release -sc)
 if [[ "$release" =~ ^(xenial|bionic|cosmic|disco|eoan|focal)$ ]] ; then
+    apt-key add "$SCRIPTS_PATH"/setup/pgdg.asc
+    cat >>$SOURCES_FILE <<EOF
```

Comment:
This needs to be `>` rather than `>>` (otherwise the file will get longer on every run and the hash will never match). The two `cat` blocks could be combined.